pedsnetdcc/abstract_transform.py

```diff
@@ -28,13 +28,16 @@ class Transform(object):
     __metaclass__ = ABCMeta
 
     @classmethod
-    def pre_transform(cls, conn_str):
+    def pre_transform(cls, conn_str, metadata=None, table_name=None):
```

Comment:
So a subclass must be prepared to be called with `table_name` either `None` or not `None`? I.e. the transform runner will call `pre_transform` once with `table_name` None and once for each table?  For instance, would `AgeTransform` do nothing unless `table_name` were `None`?  Maybe separate `pre_transform` and `pre_table_transform` methods would be easier to use if this is the case?